### DISCUSSION TITLE
【建议】添加页面收缩防遮挡功能

### DISCUSSION THREAD

**Discussion by Noah1995** [xiaoshuapp/explorer-xiaoshu#38] [Feedback]
1丶添加开关【页面收缩防遮挡】
类似[【侧边翻译】](https://github.com/EdgeTranslate/EdgeTranslate)...
(布局设置>收缩页面)
![e01f311d0953a24fb9476ef4a68198b3](https://user-images.githubusercontent.com/25548230/177725726-07752545-33a5-408f-9e60-a8a37093e697.gif)
2丶顶部加入【快捷按钮】
设置和添加分组从下面移除加入顶部...
因为如果引擎很多丶滚动距离太长操作不舒服...
上下折叠分组有人反馈我就不说了...
折叠可以加入开关触摸展开和点击展开...
![image](https://user-images.githubusercontent.com/25548230/177726438-fc12ddb2-c2e1-48ed-8aab-b71250643413.png)
还可以加入皮肤按钮一键切换黑夜和白天....
![录制_2022_07_07_16_35_44_576](https://user-images.githubusercontent.com/25548230/177730150-dcf35c47-f327-4112-bc01-ada5a35613b9.gif)
3丶顶栏下方加入自动伸缩的文字搜索框...

**Comment by nexmoe** [comment]
#39
